packages/unmock-node/src/backend/util.ts

```diff
@@ -0,0 +1,4 @@
+export const rawHeadersToHeaders = (rawHeaders: string[]) => new Array(rawHeaders.length / 2)
```

Comment:
`headersArrayToObject`?

Reply:
`rawHeaders` is a node convention that means "an array of headers".

Reply:
Yup, isn't the purpose of this function to convert an array to an object though?
Alternatively, why not use `headers` instead of `rawHeaders`?